lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -97,6 +102,10 @@ UsdMaya_ReadJob::~UsdMaya_ReadJob() { }
 
 bool UsdMaya_ReadJob::Read(std::vector<MDagPath>* addedDagPaths)
 {
+    // Do not use the global undo info recording system.
+    // The read job Undo() / redo() functions will handling all operations.
+    OpUndoInfoMuting undoMuting;
```

Comment:
The read job is an example of undo muting: it implements its own undo / redo so all undo items generated by the importer need to be discarded.